Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '>= 2.6.0', '< 3.1.0'
+ruby File.read(".ruby-version").strip
```

Comment:
This creates issues when running scripts like LSP outside the rails directory, as the path specified is not absolute.

Reply:
Is this a common pattern? Are there docs with recommendations on how to do this? I agree that we should get rid of the version range, just want to make sure we don't break tools that might be processing the Gemfile

Reply:
@tachyons good catch, as it should really use __FILE__ to address that.

@nightpool It's a fairly common pattern as you can see in these resources:

3rd Party Documentation: https://render.com/docs/ruby-version
StackOverflow recommendation by multiple people: https://stackoverflow.com/questions/32934651/is-it-a-bad-practice-to-list-ruby-version-in-both-gemfile-and-ruby-version-dotf
rbenv issue: https://github.com/rbenv/rbenv/issues/727#issuecomment-717701824